modules/nw-sriov-configuring-operator.adoc

```diff
@@ -109,6 +109,10 @@ Admission Controller application. It provides the following capabilities:
 * Mutation of the `SriovNetworkNodePolicy` CR by setting the default value for the `priority` and `deviceType` fields when the CR is created or updated.
 
 By default the SR-IOV Network Operator Admission Controller webhook is enabled by the Operator and runs as a daemon set on all control plane nodes.
+
+NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting or if you want to use unsupported devices.
```

Comment:
Minor suggestion:

You can disable the webhook under specific circumstances, for example, when troubleshooting, or if you want to use unsupported devices. 

ALTERNATIVELY, if you keep yours as-is, add a comma before "or": 

You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices.

Reply:
Thanks, I would go with my content and add the comma.